sklearn/preprocessing/_target_encoder.py

```diff
@@ -61,6 +72,9 @@ class TargetEncoder(OneToOneFeatureMixin, _BaseEncoder):
             problem. The `target_type_` attribute gives the target type used by the
             encoder.
 
+        .. versionadded:: 1.4
```

Comment:
I think `versionadded` should be reserved for adding a new `kwarg`, not an option for an existing kwarg.

```suggestion
        .. versionchanged:: 1.4
```